app/controllers/api/v2/base_controller.rb

```diff
@@ -18,7 +18,7 @@ class BaseController < Api::BaseController
 
       def_param_group :search_and_pagination do
         param :search, String, :desc => N_("filter results")
-        param :order, String, :desc => N_("Sort and order by a searchable field, e.g. 'id DESC'")
+        param :order, String, :desc => N_("Sort and order by a searchable field, e.g. '<field> DESC'")
```

Comment:
@ofedoren, @adiabramovitch I believe this is parsed by Maruku and it complains quite loudly:
```
| Maruku tells you:
+---------------------------------------------------------------------------
| Maruku cannot parse this block of HTML/XML:
| |<field> DESC'
| Missing end tag for 'field' (got 'html')
| Line: 4
| Position: 172
| Last 80 unconsumed characters:
+---------------------------------------------------------------------------
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/helpers.rb:72:in `rescue in md_html'
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/helpers.rb:67:in `md_html'
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/input/parse_span.rb:513:in `read_inline_html'
!/home/oezr/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/maruku-0.7.3/lib/maruku/input/parse_span.rb:115:in `read_span'
\___________________________________________________________________________

Wrapping bad html in a PRE with class 'markdown-html-error'
|<field> DESC'
```

Reply:
well... this is the only one I have in my develop setup currently.
Isn't there a way to escape this for Maruku not to read it as HTML tag?

Reply:
Honestly, didn't look at this yet :/ We have a redmine issue [1] to track such warnings.

[1] - https://projects.theforeman.org/issues/31889